pkg/distributors/test/MultiRewards.test.ts

```diff
@@ -69,7 +69,9 @@ describe('Staking contract', () => {
       .allowlistRewarder(pool.address, rewardToken.address, mockAssetManager.address);
 
     await expect(
-      stakingContract.connect(mockAssetManager).notifyRewardAmount(pool.address, rewardToken.address, fp(100))
+      stakingContract
+        .connect(mockAssetManager)
```

Comment:
```suggestion
        .connect(rewarder)
```

I would change the tests to use an explicitly allowlisted rewarder and change the variable name as this is more descriptive (otherwise we need to know that asset managers are privileged rewarders). We can then have a separate test for just checking that asset managers can add rewards.

Reply:
I made this rename in the other exiter PR, ok if I leave the rename out of this PR, and explicitly allowlist the user in a separate PR?

(lint inlines a lot of functions when changing `mockAssetManger` to `rewarder` and I don't want to introduce the noise to this PR, which should change `MultiRewards` minimally)

Reply:
Sure thing, this is tangential to the rest of this PR so makes sense to tackle elsewhere.